packages/design-system-docs/src/pages/index.md

```diff
@@ -29,8 +23,8 @@ The primary goals for the design system are to:
 
 In order to meet these goals we are:
 
-- Using the [U.S. Web Design Standards](https://standards.usa.gov) components and visual design as a foundation. Read more about how we're applying the Standards in our [GitHub Wiki](https://github.com/CMSgov/design-system/wiki/faq).
+- Using the [U.S. Web Design System](https://designsystem.digital.gov/) components and visual design as a foundation.
 - Reducing the overall footprint of websites by writing [scalable CSS]({{root}}/guidelines/code-conventions).
-- Supporting rapid development with [utility classes]({{root}}/utilities) and easy-to-use components with best practices baked in.
+- Supporting rapid development with utility classes and easy-to-use components with best practices baked in.
```

Comment:
why was this link removed?

Reply:
Oversight on my part - `utilities/` does not exist anymore and it needed to be changed to `utilities/overview/`